docs/SetupDatabaseForAWS.md

```diff
@@ -21,3 +21,12 @@ Note:-
 
 * Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_CreateDBInstance.html) to create an AWS RDS MySQL/PostgreSQL/Oracle/SQL Server database instance.
 * (Optional) Configure advanced monitoring services with the [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Monitoring.html), monitoring is enabled by default on AWS RDS database instances.
+
+
+## Amazon Aurora for MySQL/PostgreSQL
+
+### Steps
+
+* Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html) to create an Amazon Aurora cluster for MySQL/PostgreSQL.
+* (Optional) Follow this [AWS official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Managing.Performance.html) to manage performance and scaling for Aurora DB clusters and DB instances.
```

Comment:
Is there any specific reason for adding performance and scaling steps for Aurora?